Libraries/Connect/Interfaces/ConnectError.swift

```diff
@@ -73,6 +73,23 @@ public struct ConnectError: Swift.Error, Sendable {
             case payload = "value"
         }
 
+        public init(from decoder: Swift.Decoder) throws {
+            let container = try decoder.container(keyedBy: CodingKeys.self)
+
+            // Read the base64-encoded payload and then pad if needed
+            let encodedPayload = try container.decode(String.self, forKey: .payload)
+            let padded = encodedPayload.padding(
+                toLength: ((encodedPayload.count+3)/4)*4,
+                withPad: "=",
+                startingAt: 0
+            )
+
+            self.init(
+                type: try container.decodeIfPresent(String.self, forKey: .type) ?? "",
+                payload: Data(base64Encoded: padded)
+            )
+        }
+
```

Comment:
@rebello95 Here is the padding fix I mentioned.

Reply:
Do tests cover receiving both a padded and unpadded error?

Reply:
Do you mean the conformance suite? If so, no, they don't account for that. But probably a good one to add when it's refactored.